app/models/user.rb

```diff
@@ -1,10 +1,23 @@
 class User < ActiveRecord::Base
   validates :full_name, presence: true
-  validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i }
-  validates :display_name, length: { maximum: 32, minimum: 2 }, presence: false
+  validates :email,
+             format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i }
```

Comment:
Since this a long regex, if you break the fields 63 test fail. Not sure how to fix that, via a new line.